src/common/transport/transport-client.js

```diff
@@ -0,0 +1,80 @@
+import { TransportClientError } from '../errors';
+import rp from 'request-promise';
```

Comment:
I'm not sure if Domo APIs support CORS yet but I think enabling that will help us get people up and running much faster!